src/cryptography/x509/base.py

```diff
@@ -436,6 +436,11 @@ def not_valid_before(self, time):
         if time <= _UNIX_EPOCH:
             raise ValueError('The not valid before date must be after the unix'
                              ' epoch (1970 January 1).')
+        if self._not_valid_after is not None and time > self._not_valid_after:
```

Comment:
Yeah, I decided equality is okay since a cert valid for 1s isn't out of spec (that's why the commit notes `>=`). The ValueError doesn't tell you that, but we could just say "the not valid before date must be <= the not valid after date" I suppose? I'm neutral on changing it, so if you want to let me know.